src/rules/completedDocsRule.ts

```diff
@@ -405,19 +422,27 @@ class CompletedDocsWalker extends Lint.ProgramAwareRuleWalker {
         }
 
         const comments = symbol.getDocumentationComment();
-        this.checkComments(node, nodeType, comments);
+        this.checkComments(node, this.describeNode(nodeType), comments, requirementNode);
+    }
+
+    private describeNode(nodeType: DocType): string {
```

Comment:
Seems a little hardcody to just do this to `ARGUMENT_ENUM_MEMBERS`. How about generalizing the logic by taking in both `node` and `requirementNode`? If `node` is an ancestor of `requirementNode`, append `" members"` to the description.